chart/kubeseal-webgui/templates/deployment.yaml

```diff
@@ -20,9 +20,7 @@ spec:
       securityContext:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      {{- if .Values.serviceaccount.create }}
-      serviceAccountName: kubeseal-webgui
-      {{- end }}
+      serviceAccountName: {{ default "kubeseal-webgui" .Values.customServiceAccountName }}
```

Comment:
I like the order `{{ .Values.customServiceAccountName | default "kubeseal-webgui" }}` better :) and maybe we could add a `quote` to be on the safe side?